pkg/oc/bootstrap/docker/run_self_hosted.go

```diff
@@ -183,7 +183,8 @@ func (c *ClusterUpConfig) StartSelfHosted(out io.Writer) error {
 		return err
 	}
 
-	installContext, err := componentinstall.NewComponentInstallContext(c.openshiftImage(), c.imageFormat(), c.BaseDir, c.ServerLogLevel)
+	installContext, err := componentinstall.NewComponentInstallContext(c.openshiftImage(), c.imageFormat(), c.defaultPullPolicy, c.BaseDir,
```

Comment:
defaultPullPolicy should be renamed `pullPolicy` it's not a default, it's the value.

Reply:
agree, renamed.